js/ui/scheduler/workspaces/view_model/view_data_generator_month.js

```diff
@@ -85,9 +85,18 @@ export class ViewDataGeneratorMonth extends ViewDataGenerator {
     }
 
     getRowCount(options) {
```

Comment:
Minor comment.

Let's deconstruct the options:
```suggestion
    getRowCount({currentDate, intervalCount, firstDayOfWeek}) {
```

Reply:
it would look bad after typing this function: 
```typescript
    getRowCount({currentDate, intervalCount, firstDayOfWeek}: {currentDate: Date, intervalCount: number, firstDayOfWeek: number}) {
```